geolocator/lib/geolocator.dart

```diff
@@ -261,6 +261,31 @@ class Geolocator {
         timeLimit: timeLimit,
       );
 
+  /// Fires every time NMEA-0183 sentences are received.
+  ///
+  /// This event starts all location sensors on the device and will keep them
+  /// active until you cancel listening to the stream or when the application
+  /// is killed.
+  ///
+  /// ```
+  /// StreamSubscription<NmeaMessage> nmeaStream = getNmeaStream()
+  ///     .listen((NmeaMessage nmea) {
+  ///       // Handle NMEA changes
+  ///     });
```

Comment:
Code example should include static class: 
```suggestion
  /// StreamSubscription<NmeaMessage> nmeaStream = Geolocator.getNmeaStream()
  ///     .listen((NmeaMessage nmea) {
  ///       // Handle NMEA changes
  ///     });
```